src/main/kotlin/no/nav/familie/ef/sak/beregning/OmregningService.kt

```diff
@@ -86,14 +87,20 @@ class OmregningService(private val behandlingService: BehandlingService,
         val innvilgelseOvergangsstønad =
                 vedtakHistorikkService.hentVedtakForOvergangsstønadFraDato(fagsakId,
                                                                            YearMonth.from(nyesteGrunnbeløpGyldigFraOgMed))
-
+        feilHvis(innvilgelseOvergangsstønad.samordningsfradragType != null && samordningsfradrag == null) {
```

Comment:
Hvordan ønsker vi å behandle disse? 
Alle med samordningsfradrag kommer feile, og vi må då avvikshåndtere de? 

Burde vi i stedet kaste en annen type exception, fange den i tasken og bare logge fagsakId for denne eller noe slik? Sånn att vi kan legge ut de i en liste og ikke trenger å avvikshåndtere alle tasker?

Reply:
Det er en mulighet. Vi snakket om at det dreide seg om ca. 67 fagsaker, så jeg tenkte at 67 feilede tasker var greitr som arbeidsliste. Hvis du synes vi bar skal logge dem, så er det helt greit for min del. Har du noen innspill til hvordan en slik loghgmelding burde se ut?